app/helpers/application_helper.rb

```diff
@@ -3,4 +3,12 @@ def gravatar_url(email)
     gravatar_hash = Digest::MD5.hexdigest(email).downcase
     "http://gravatar.com/avatar/#{gravatar_hash}.png"
   end
+
+  def class_if?(klass, exp)
+    exp ? klass : nil
```

Comment:
I know I wrote this haha but let's rename `klass` to `css_class` so it's more clear what that is supposed to be a CSS class.